frontend/src/hooks/analytics/use-analytics-track-on-mount.ts

```diff
@@ -0,0 +1,8 @@
+import { Dict } from "mixpanel-browser";
+import { useEffect } from "react";
+import { useAnalyticsTrack } from "./use-analytics-track";
+
+export const useAnalyticsTrackOnMount = (event: string, args?: Dict) => {
```

Comment:
If I'm not mistaken we have to use `useAnalyticsTrackOnMount` on pages only? So, maybe it's better to provide an explanation above?

Reply:
Well it depends on our goals, we may want to log not the page mount, but specific component.
At the moment it is true, but one can easily find it out by looking up hook usage.